adapters/melozen/melozentest/exemplary/app-video.json

```diff
@@ -0,0 +1,137 @@
+{
```

Comment:
as per `static/bidder-info/melozen.yaml` video media type is not supported for app.

should update bidder info yaml to support video media type

Reply:
Thank you for pointing out the omission of video media type support in the yaml configuration for both site and app. I have updated the bidder info in the yaml file to include video as a supported media type and have also added corresponding JSON tests (app-video.json and web-video.json) to ensure coverage. The PR has been updated accordingly. Please review the changes at your convenience. Thanks again for your guidance!